authorization/src/main/java/com/terabite/authorization/controller/AuthorizationController.java

```diff
@@ -34,4 +41,16 @@ public Payload userLoginPost() {
     public Payload userLogoutPost() {
         return new Payload("Reached logout POST");
     }
+    
+    @PutMapping("/forgot_password")
+    @ResponseStatus(HttpStatus.OK)
+    public void forgotPassword(@RequestBody String jsonEmail) {
+        forgotPasswordHelper.processForgotPassword(jsonEmail);
+    }
+
+    @PutMapping("/reset_password")
+    @ResponseStatus(HttpStatus.OK)
+    public void resetPassword(@RequestParam String token, @RequestBody String jsonPassword) throws JsonProcessingException, LoginNotFoundException{
+        forgotPasswordHelper.processResetPassword(token, jsonPassword);
+    }
```

Comment:
Same thing as above. Make it clear when something goes wrong so it's easier for frontend to work with this.

Reply:
Done, 400 for bad request, 200 for good with a message for both.